support-frontend/assets/helpers/forms/stripe.ts

```diff
@@ -40,24 +41,25 @@ export interface StripeKey {
 function getStripeKey(
 	stripeAccount: StripeAccount,
 	country: IsoCountry,
+  currency: IsoCurrency,
 	isTestUser: boolean,
 ): string {
-	switch (country) {
-		case 'AU':
-			return isTestUser
-				? window.guardian.stripeKeyAustralia[stripeAccount].test
-				: window.guardian.stripeKeyAustralia[stripeAccount].default;
-
-		case 'US':
-			return isTestUser
-				? window.guardian.stripeKeyUnitedStates[stripeAccount].test
-				: window.guardian.stripeKeyUnitedStates[stripeAccount].default;
-
+  let account;
+  switch (currency) {
+		case 'AUD': // need to match with how support-workers does it
+      account = window.guardian.stripeKeyAustralia[stripeAccount];
+      break;
+		case 'USD':
+      if (country === 'US') { // this allows support of US only cards (for single)
+        account = window.guardian.stripeKeyUnitedStates[stripeAccount];
+      } else {
+        account = window.guardian.stripeKeyDefaultCurrencies[stripeAccount];
+      }
```

Comment:
we tried to do this as a fallthough case, but that was red underlined by typescript, which is fair enough